Simplenote/src/androidTest/java/com/automattic/simplenote/screenshots/ScreenshotTest.java

```diff
@@ -126,20 +127,15 @@ public void screenshotTest() throws InterruptedException {
             waitForViewMatching(allOf(withId(R.id.note_title), withText(title)), 5000);
         }
 
-        selectNoteFromNotesList(NOTE_FOR_EDITOR_SHOT_TITLE);
+        selectNoteAndTakeScreenshotFromNotesList(NOTE_FOR_EDITOR_SHOT_TITLE, "01-note");
 
-        // It can happen that the email verification screen appears on the note editor instead of
-        // the note list screen, so look for one and dismiss it if found.
-        dismissVerifyEmailScreenIfNeeded();
-
-        Screengrab.screenshot("01-note");
-
-        dismissNoteEditor();
```

Comment:
I dried all the code (red) in the new method (green) because the shot for the note with the internal links uses the same sequence.